src/main.js

```diff
@@ -218,6 +218,13 @@ allowProtocolDialog.init(mainWindow);
 // This method will be called when Electron has finished
 // initialization and is ready to create browser windows.
 app.on('ready', function() {
+  ipcMain.on('notified', function(event, arg) {
```

Comment:
Take that with a grain of salt, as I haven't tested it and I'm not a node/js expert.